data/field.go

```diff
@@ -38,15 +39,24 @@ type Fields []*Field
 // NewField returns a instance of *Field. Supported types for values are:
 //
 // Integers:
-//  []int8, []*int8, []int16, []*int16, []int32, []*int32, []int64, []*int64
+//
```

Comment:
In an effort to keep the "real" PR smaller and more legible, this is batching all the auto-formatting updates into this PR 😬